src/components/notification-icon.component.js

```diff
@@ -1,19 +1,33 @@
 // @flow
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
-import { View } from 'react-native';
+import { View, StyleSheet } from 'react-native';
 import { Icon } from 'react-native-elements';
 
 import { Badge } from 'components';
 
+import { colors } from 'config';
+
+const styles = StyleSheet.create({
+  icon: {
```

Comment:
This is still my fault, but we will not have alignment, without `flex: 1`, so it can move the styles to the `badgeContainer`? Though all the same the vertical alignment not absolutely on the center seems ..

Reply:
You're right, adding `flex: 1` messes up positioning entirely so removed this style completely and it looks fine without :+1: